From c41c695ee66fbb54c16c19d4093606e0cf0735be Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Tue, 28 Jun 2022 05:38:09 +0530
Subject: [PATCH] Remove C99 flag

Removed c99 flag to fix below build error:
error: "BoringSSL must be built in C11 mode or higher."

As per commit in external/boringssl:
"Try to require C11 (in non-MSVC compilers).
Update-Note: If the build fails with an error about C11,
remove -std=c99 or -std=gnu99 from your build.
Refcounting will get faster"

Signed-off-by: svenate <salini.venate@intel.com>
---
 libsslsupport/Android.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libsslsupport/Android.mk b/libsslsupport/Android.mk
index 19d59b6..cfe3afd 100644
--- a/libsslsupport/Android.mk
+++ b/libsslsupport/Android.mk
@@ -131,7 +131,6 @@ KF_SSL_PATH := $(KERNELFLINGER_SSLSUPPORT_PATH)/borningssl
 $(shell cp bionic/libc/include/stdio.h $(KF_SSL_PATH))
 $(shell sed -i 's/__sF\[\] __REMOVED_IN(23)/__sF\[\]/g' $(KF_SSL_PATH)/stdio.h)
 LOCAL_CFLAGS += -I$(KF_SSL_PATH)
-LOCAL_CFLAGS += -std=c99
 LOCAL_CFLAGS += -I$(LOCAL_PATH)/include
 LOCAL_CFLAGS += -DOPENSSL_NO_THREADS_CORRUPT_MEMORY_AND_LEAK_SECRETS_IF_THREADED
 LOCAL_CFLAGS += -D__ANDROID_API__=21
-- 
2.17.1

